github-sync/github-data/users.yaml

```diff
@@ -159,9 +160,6 @@ users:
         role: maintainer
   - username: efebarlas
```

Comment:
same question here
@haydentherapper @bobcallaway by removing the user from that team, meaning they don't have any other contribution for sigstore? should we keep the user as well or remove it from the org as well?

Reply:
I think we can remove efe

Reply:
can you approve this pr @SantiagoTorres :)
i will do the removal in a followup